spec/helpers/provider_helper_spec.rb

```diff
@@ -1,15 +1,42 @@
 require 'rails_helper'
 
 feature 'View helpers', type: :helper do
+  let(:email) { 'ab@c.com' }
+  let(:provider) { jsonapi(:provider).to_resource }
+
   describe "#add_course_link" do
```

Comment:
If there is this test should there not be add_location_link one? 

Seeing That _blank thing...

Reply:
fixed